apps/custom-ui-floating-window-demo/components/FollowUp/ActionButtons.js

```diff
@@ -0,0 +1,68 @@
+import {
+  initializeSDK,
+  openActivityModal,
+  redirectToContact,
+} from '../../shared/custom_ui_sdk';
+import logger from '../../shared/logger';
+const log = logger('FollowUp');
+
+const FollowUpActionButtons = (props) => {
+  // Invokes the Activity Modal
+  const invokeActivityModal = async () => {
+    log.info('Invoking activity dialog...');
+    const sdk = await initializeSDK();
+    openActivityModal(sdk);
+  };
+  // Navigates to the newly created contact in Pipedrive
+  const goToContact = async () => {
+    log.info('Navigating to the contact based on the contact ID');
+    const sdk = await initializeSDK();
+    redirectToContact(sdk, props.callerDetails.id);
```

Comment:
SDK should only be initialised once (it doesn't break anything but doing it this way is kinda anti-pattern). Can you initialise it somewhere once (maybe during initial load) and re-use that sdk variable across your app?

Reply:
Good point. Initializing it once.